src/SyntheticGrids.jl

```diff
@@ -73,8 +79,10 @@ load_grid
 import Base.==
 import Base.show
 import Base.hash
-import Base.mean
 import Base.merge!
+using Random: seed!
+using Statistics: Statistics, mean
```

Comment:
Should be combined with the above `using` calls. Ideally you only call `using Statistics` once